### PR TITLE
Add Meetup checklist.

### PR DESCRIPTION
**Pull request by ryan-roemer**
As we've had a few bumps in the past over the details, here's an attempt at a checklist to serve as a template for each meetup for us to ensure we all provide an awesome time for our Node.js community!

Comments, additions, suggestions, etc. most welcome.

/cc @hamoore @scottnonnenberg @jasonwilson @ryanisinallofus @ggoodale

### CODE REVIEW

File: meetup-checklist.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+* [ ] Get a venue host.
+    * [ ] Confirm **maximum seating capacity** and enter in meetup RSVP limit.
+
+* [ ] Get a food / beverages host _or_ confirm venue host will provide.
+    * [ ] Confirm _what_ is actually going to be served (e.g., snacks vs. dinner).
+
+* [ ] Get first speaker.
+    * [ ] Get title / abstract / bio / twitter handle.
+    * [ ] Confirm talk length (20-30 mins) and limiting commercial content.
```
Comment (jasonwilson) [inline meetup-checklist.md:21]: I can't stress this enough: "limiting commercial content". The last meetup felt like an advertisement for surface and azure!

Comment (scottnonnenberg) [reply meetup-checklist.md:21]: Agree. A tiny, tiny bit of code was presented. :0(

File: meetup-checklist.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      * [ ] Body: Paste HTML from WIP GitHub post. Then check it all rendered
+        correctly.
+            * [ ] Remove the `XXX` `meetup` link from the post.
+      * [ ] RSVP settings:
+          * [ ] Set a maximum number of RSVPs
+          * [ ] Uncheck `Allow members to go with up to __ guests` button.
+            (We want all our attendees to be meetup members.)
+      * [ ] Click the `Schedule this meetup now`
+      * [ ] Also do the email members option presented after scheduling the
+        meetup.
```
Comment (scottnonnenberg) [inline meetup-checklist.md]: These nested lists aren't rendering properly on github. :0(

File: meetup-checklist.md (hunk 1 above)
Comment (jasonwilson) [reply meetup-checklist.md:21]: literally copy pasted connection strings 😢

File: meetup-checklist.md (hunk 2 above)
Comment (ryan-roemer) [reply meetup-checklist.md]: Ugh. Yeah. I'll figure it out.

Comment (ryan-roemer) [reply meetup-checklist.md]: OK, I think I've fixed the indentation thing now.

**Comment by ryan-roemer** [comment]
I think I've addressed all the comments. Last chance for review and thanks!

**Pull request merged**